test/self-hosted-runner/env.in

```diff
@@ -12,8 +12,8 @@ GH_APP_PRIVATE_KEY_FILE=""
 GH_REPO=""
 
 PROXY=""
-HTTP_PROXY="http://${PROXY}"
-HTTPS_PROXY="http://${PROXY}"
+HTTP_PROXY="${PROXY:+http://}${PROXY}"
+HTTPS_PROXY="${PROXY:+http://}${PROXY}"
```

Comment:
Hmm... true. And I was thinking to do the substitution in `make create-env|env`, but I left it as a pure `cp`, so it can't work. Back to the drawing board.